packages/augur-ui/src/modules/trading/components/form/form.styles.less

```diff
@@ -1,81 +1,83 @@
 @import (reference) "~assets/styles/shared";
 
-.TradingForm__form-body {
-  padding: 0.75rem 1rem 0;
+.Form {
+  > ul {
+  padding: @size-12 @size-16 0;
 
-  > li {
-    color: @color-secondary-text;
-    display: flex;
-    flex-flow: column nowrap;
-    font-size: @font-size-normal;
-    width: 100%;
+    > li {
+      color: @color-secondary-text;
+      display: flex;
+      flex-flow: column nowrap;
+      font-size: @font-size-normal;
```

Comment:
remove this and use `.text` here, also if you see any other `@font-size-normal` replace them with new variables and or use `.text/.mono` versions for font specific things